spec/Memory_Consistency.tex

```diff
@@ -66,29 +71,29 @@ \section{Design Principles}
 
 \section{Overview of the Memory Consistency Model}
 
-The memory consistency model for Chapel is the same as the C11 and C++11 memory
+The memory consistency model for Chapel is similar to the C11 and C++11 memory
 consistency models, with some minor exceptions which are described in section
 \ref{relating_to_C_MCM}. Section \ref{SC_for_DRF} directly describes Sequential
-Consistency for Data Race Free programs as an alternative way of understanding
+Consistency for Data-Race-Free programs as an alternative way of understanding
 the recommended subset of the C11 and C++11 memory consistency models. This
 description does not fully specify the behavior of atomics using a more relaxed
-ordering than sequential consistency. While those features are available in
-Chapel, they are not recommended casual use since it difficult to understand
-how to write correct programs. Section \ref{relating_to_C_MCM} describes how to
-use the C11 and C++11 specifications to understand the behavior of Chapel
-programs using these more relaxed features.
+ordering than sequential consistency. While relaxed atomic operations are
+available in Chapel, their use is highly discouraged. For advanced users,
+section \ref{relating_to_C_MCM} describes how to use the C11 and C++11
+specifications to understand the behavior of Chapel programs using relaxed
+atomic operations.
```

Comment:
I'd prefer for this section to be a little bit pessimistic - to at least summarize what we mean by "highly discouraged". Can we say "they should be reserved for advanced users only" or something like that? And say that the problem is that the program can become very confusing?